model/project_ref.go

```diff
@@ -553,6 +553,38 @@ func (p *ProjectRef) DetachFromRepo(u *user.DBUser) error {
 	return catcher.Resolve()
 }
 
+func (p *ProjectRef) ChangeOwnerRepo(u *user.DBUser) error {
```

Comment:
The way I thought this function would work would be to take in a project, and take in the the repo we want it to be under, and change that project to now be associated with the new repo. This only takes in a user, and I'm a bit confused about how a graphql resolver would call this.